### PR TITLE
http-parser: fix gcc-7 build

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

File: components/library/http-parser/patches/02-gcc7.patch [hunk 1]
```diff
@@ -0,0 +1,19 @@
+--- http-parser-2.6.2/http_parser.c.orig	2018-02-13 15:44:16.533837129 +0000
++++ http-parser-2.6.2/http_parser.c	2018-02-13 15:45:21.539702963 +0000
+@@ -1812,6 +1812,7 @@
+             case 0:
+               break;
+ 
++            /* FALLTHROUGH */
```
Comment (pyhalov) [inline components/library/http-parser/patches/02-gcc7.patch]: Why? We have break here.

Comment (alarcher) [reply components/library/http-parser/patches/02-gcc7.patch]: Just imported upstream patch without thinking...

**Comment by alarcher** [comment]
@pyhalov Fixed.

**Review by pyhalov — APPROVED** [review]

**Comment by pyhalov** [comment]
ln: cannot create /jenkins/jobs/oi-userland/workspace/components/library/http-parser/build/prototype/i386//usr/lib/libhttp_parser.so: File exists

**Pull request merged**